Source/Tools/FEXLoader/FEXLoader.cpp

```diff
@@ -404,10 +403,14 @@ int main(int argc, char** argv, char** const envp) {
   FEXCore::Profiler::Init();
   FEXCore::Telemetry::Initialize();
 
-  RootFSRedirect(&Program.ProgramPath, LDPath());
-  InterpreterHandler(&Program.ProgramPath, LDPath(), &Args);
+  // Program.ProgramPath is highly likely to be prefixed with FEX's rootfs. Get rid of that.
+  if (Program.ProgramPath.starts_with(LDPath())) {
+    Program.ProgramPath = Program.ProgramPath.substr(LDPath().size());
+  }
```

Comment:
Is there any chance this will result in a relative path? If not, how is it prevented? It would probably be undesirable behavior.

Reply:
Is there a chance we accidentally strip one '/' off to many, though? For example, does it make a difference if the user configures the RootFS as "/home/user/rootfsfolder/" vs "/home/user/rootfsfolder"?

Another possible failure path: Can the program paths end up containing "..", escaping the RootFS?

Reply:
To clarify, I'm not concerned about LDPath being relative, but about the result of calling `substr()`.

Reply:
> Another possible failure path: Can the program paths end up containing "..", escaping the RootFS?

Probably, but this doesn't change any behaviour around that anyway. It'll be broken before and it'll be broken afterwards.

Reply:
Slightly changed the code doing some more thorough testing, to ensure the beginning `/` isn't stripped off.